Add support for Django 3.2

con-cat opened the pull request:
- Configure default auto field for waffle and test_app - Django 3.2 adds
  the ability to customise the type of auto-created primary key fields.
- Add Django 3.2 to tox.ini and setup.py

Addresses #395
Fixes #396

con-cat committed:
Add support for Django 3.2

- Configure default auto field for waffle and test_app - Django 3.2 adds
  the ability to customise the type of auto-created primary key fields.
- Add Django 3.2 to tox.ini and setup.py

con-cat committed:
Update deprecated django-admin.py entrypoint in run.sh

This commit changes this to just django-admin, as django-admin.py will
be deprecated in Django 4.0

MaybeThisIsRu commented on waffle/apps.py:
```diff
@@ -4,6 +4,7 @@
 class WaffleConfig(AppConfig):
     name = 'waffle'
     verbose_name = 'django-waffle'
+    default_auto_field = 'django.db.models.AutoField'
```
If I install `django-waffle` (once released with this change) and install it on Django 3.2, would it be in my interest to use `BigAutoField` even for this package, by manually configuring this?

Sorry if I'm getting something obvious very wrong here.

con-cat replied:
You shouldn't need to change this - `AutoField` is still the default for projects that have been upgraded to 3.2, and the difference between the two is pretty minimal aside from `BigAutoField` giving a larger range of values when you get to very high ID numbers.

I left it as `AutoField` to keep things simple and avoid generating migrations, but I'd be happy to change to `BigAutoField` if people would prefer.

MaybeThisIsRu replied:
That makes sense. I should have clarified I'm asking from the point of view of a fresh Django 3.2 project, rather than an upgrade.

con-cat replied:
You shouldn't need to change it either way

con-cat committed:
Remove Django main branch from tox.ini envlist

Tests are currently failing against main because of some Django 4.0
compatibility issues
